ensembl/htdocs/info/genome/variation/prediction/protein_function.html

```diff
@@ -125,6 +138,7 @@ <h2 id="sift">SIFT</h2>
 
 <br />
 
+<h2>Results only available for human variants</h2>
```

Comment:
Should we add more vertical separation above ? Or an horizontal bar ?

Reply:
I've not done this, but feel free to beautify later!